Healex.HL7v2Anonymizer/appsettings.json

```diff
@@ -70,75 +70,75 @@
                     },
                     {
                         "Path": "PID.11.5",
-                        "Value": "Zip code"
+                        "Value": "A104"
                     },
                     {
                         "Path": "PID.11.6",
                         "Value": "Country"
```

Comment:
v2 specification specifies an ID of max length 3 letter code here 'DEU' or so (https://hl7-definition.caristix.com/v2/HL7v2.5/Fields/PID.11.6)

Reply:
Awesome, thank you!

Reply:
Is this about PID-11.5 zip code or the PID-11.6 country? I think you changed the wrong value there and "Country" should be changed.